From f143d49f1d9ca2bb7058d7e09c331d66ee86f168 Mon Sep 17 00:00:00 2001
From: Arun Easi <arun.easi@qlogic.com>
Date: Wed, 30 Jan 2013 03:34:40 -0500
Subject: [PATCH 344/858] [SCSI] qla2xxx: Allow ISP81xx to create ATIO queues.

Signed-off-by: Arun Easi <arun.easi@qlogic.com>
Signed-off-by: Saurav Kashyap <saurav.kashyap@qlogic.com>
Signed-off-by: James Bottomley <JBottomley@Parallels.com>
(cherry picked from commit b8aa4bdfe13ac3ff51af5d8f79ae0adc20405215)

Signed-off-by: Clark Williams <williams@redhat.com>
Signed-off-by: John Kacur <jkacur@redhat.com>
---
 drivers/scsi/qla2xxx/qla_os.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/scsi/qla2xxx/qla_os.c b/drivers/scsi/qla2xxx/qla_os.c
index e93408f..2d2afdb 100644
--- a/drivers/scsi/qla2xxx/qla_os.c
+++ b/drivers/scsi/qla2xxx/qla_os.c
@@ -2425,6 +2425,7 @@ qla2x00_probe_one(struct pci_dev *pdev, const struct pci_device_id *id)
   ha->mbx_count = MAILBOX_REGISTER_COUNT;
   req_length = REQUEST_ENTRY_CNT_24XX;
   rsp_length = RESPONSE_ENTRY_CNT_2300;
+  ha->tgt.atio_q_length = ATIO_ENTRY_CNT_24XX;
   ha->max_loop_id = SNS_LAST_LOOP_ID_2300;
   ha->init_cb_size = sizeof(struct mid_init_cb_81xx);
   ha->gid_list_info_size = 8;
-- 
1.8.3.1